[2024-09-23T18:07:40.499+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-09-23T18:07:38.926845+00:00 [queued]>
[2024-09-23T18:07:40.517+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-09-23T18:07:38.926845+00:00 [queued]>
[2024-09-23T18:07:40.519+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-09-23T18:07:40.520+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-09-23T18:07:40.521+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-09-23T18:07:40.545+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-09-23 18:07:38.926845+00:00
[2024-09-23T18:07:40.553+0000] {standard_task_runner.py:55} INFO - Started process 14676 to run task
[2024-09-23T18:07:40.559+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2024-09-23T18:07:38.926845+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp5gjyvfyx']
[2024-09-23T18:07:40.563+0000] {standard_task_runner.py:83} INFO - Job 44: Subtask is_api_available
[2024-09-23T18:07:40.673+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2024-09-23T18:07:38.926845+00:00 [running]> on host 3faff16bae04
[2024-09-23T18:07:40.786+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-09-23T18:07:38.926845+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-09-23T18:07:38.926845+00:00
[2024-09-23T18:07:40.789+0000] {http.py:120} INFO - Poking: api/
[2024-09-23T18:07:40.814+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-09-23T18:07:40.818+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-09-23T18:07:41.189+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-09-23T18:07:41.209+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20240923T180738, start_date=20240923T180740, end_date=20240923T180741
[2024-09-23T18:07:41.256+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-09-23T18:07:41.294+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
